content/articles/wand-library-in-python/index.md

```diff
@@ -0,0 +1,151 @@
+    print('The Height of the image is:', img.height)
+    print('The width of the image is:', img.width)
+```
+- The first step is to import the image library from the Wand.
+-  The second step in this process is to import an image that you want to get its details.
+- However, to ensure that the image details will be output successfully, the correct pathname should be input and in the correct format. Furthermore, this mainly applies to an image located within several folders. The following format should be used when encountering such an image.
+-  The third step is now to print the details of the image, i.e., the width and the height.
+
+The input image is:
+
+![python](/engineering-education/wand-library-in-python/python.png)
+
+The output is as follows.
+```bash
+The height of the image is: 203
+The width of the image is: 601
+```
+### Use Wand package in Python to blur an image
+We can use several ways in Python to blur an image. For example, one can decide to blur an image using the pillow package, another one can abandon the pillow package and use the OpenCV library, and all will perform the same job. However, in our case, we will use the Wand library to blur the same image, and the following are the steps that we shall use.
+
+Let us take a look at an example.
+```Python
+  # Importing the image library from Wand
+from Wand.image import Image as wandImage
+# Importing the image using the correct image location.
+with wandImage(filename='/wand-library-in-python/python.png') as img:
+       # Blur function and its dimensions    
+    img.blur(radius=1, sigma=3)
+      # Saving the blurred image using in a certain location.    
+    img.save(='/wand-library-in-python/python1.png)
+```
+- We shall import the image library from Wand as we had done in our previous example.
+- We shall then import our image using the correct pathname.
+- We shall then call the blur function by specifying the radius and sigma of the image. The radius parameter is used to determine the radius of the gaussian aperture, which is the size of the gap, and it is always an integer number, e.g., 3. The sigma parameter is used to specify the value of sigma, which is the Gaussian filter's standard deviation, and it is also an integer input, e.g., 5.
+- The last step is to save the image from differentiating it from the original image.
+
+The input image is:
+
+![python](/engineering-education/wand-library-in-python/python.png)
+
+The output image is:
+
+![python](/engineering-education/wand-library-in-python/python1.png)
+### Image transform using Python wand
+Another essential and robust feature of the Python wand is the image transform feature. It is used to transform an image to be displayed differently from the display of the original image. Image transformation, in this case, means that the output of the image will be upsidedown and reversed as compared to the input image. The image transformation is achieved through the collaboration of several functions. 
```

Comment:
Image transform is a more generic term.

You need to be specific - like rotating image or flipping the image upside down and so on.